### ISSUE TITLE
feat(patch): Hide/Skip videos on Autoplay

### ISSUE THREAD

**Issue by minhperry** [revanced/revanced-patches#1976]
### Application

YT Revanced

### Issue

Random unrelated videos in an autoplay streak

### Patch

There should be an option or button somewhere that will add a video to a list, and if autoplay is on whenever it reaches that video, or have that as next video, it will either instantly skip or just don't play that one at all.

### Motivation

I'm a rSlash enjoyer and I usually just put YT on autoplay while it switches between video. But once in a while it just insert TomSka's "Meanwhile 4D" out of nowhere. This has happened to me SO MUCH OVER AND OVER AGAIN to a point that the video is annoying and not even funny anymore. I've also read the comments of that video and people are also complaining about that and that one only ruining their autoplay experience.

### Acknowledgements

- [X] I have searched the existing issues and this is a new and no duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I filled out all of the requested information in this issue properly.

**Comment by minhperry** [comment]
> You can tell YouTube to not show the video again.

Yes, I did that every time it ruins my autoplay and yet the next time it will continue to ruin

**Comment by oSumAtrIX** [comment]
This is an issue with YouTube. Contact support.
